ByteBrewers12/govschemes
# Feature: Multilingual Support in Angular App

Pull Request Description:
🌐 Multilingual Support in Angular App

Changes Made:
Angular App:

Implemented a robust and user-friendly language feature to enhance the user experience.
Integrated language switch functionality in the UI for seamless navigation.
Ensured responsive design to accommodate various language text lengths.
Backend Integration:

Extended backend APIs to support language preferences during data retrieval.
Enhanced data fetching logic to dynamically serve content in the selected language.
Implemented language-specific data handling to ensure a consistent user experience.
Additional Details:
Languages Supported: English, Spanish, French, German, and Italian.
Backend API Endpoints:
/api/data - Fetches multilingual content based on user language preference.
/api/languages - Retrieves the list of supported languages.
Testing Instructions:
Frontend:

Navigate to the app.
Use the language switcher to toggle between supported languages.
Verify that UI elements adapt to the selected language.
Backend:

Test API endpoints /api/data and /api/languages.
Ensure the content retrieved corresponds to the selected language.
